base/expr.jl

```diff
@@ -88,9 +88,9 @@ julia> macro m()
 julia> M.f()
 (1,2)
 ```
-With @macroexpand the expression expands where @macroexpand appears in the code (module M).
-With macroexpand the expressions expands in the current module where the code was finally called.
-Note that when calling macroexpand or @macroexpand directly from the REPL, both of these contexts coincide, hence there is no difference.
+With `@macroexpand` the expression expands where `@macroexpand` appears in the code (module `M`).
+With `macroexpand` the expressions expands in the current module where the code was finally called (REPL).
```

Comment:
Let me explain it differently. If you call `macroexpand`, the expansion takes place at runtime. If you call `@macroexpand` the expansion takes place at compile time (when `@macroexpand` is expanded to be more precise.). So the context of `@macroexpand` is where it appears in the code, while the context of `macroexpand` is the current environment. So if you run M.f() from a script, the answer will be 
`(1, whatever @m() currently means in your script)`
Does this answer the question?